R/markov_model.R

```diff
@@ -947,19 +955,18 @@ markov_model <- R6Class("markov_model", list(
     }
     
     names(df_results) <- c("Result", self$v_treatment_names)
-    
-    writeData(output_wb, 
-              sheet = "Results", 
-              x = df_results,
-              startCol = startCol, 
-              startRow = startRow)
-    
-    
-    
-    return_value <- saveWorkbook(output_wb, 
-                                 file = wb_filename, 
-                                 overwrite = TRUE,
-                                 returnValue = TRUE)
+
+    wb$add_data(
+      sheet = "Results",
+      x = df_results,
+      start_row = startRow,
+      start_col = startCol,
+      na.strings = NULL
+    )
+
+    wb$save(file = wb_filename)
+    # returnValue
+    file.exists(wb_filename)
```

Comment:
`file.exists()` should always be true, since the file is loaded in the first place, but I wanted something that keeps the old logic